ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -233,46 +233,129 @@ class SignalProducerSpec: QuickSpec {
 		}
 
 		describe("first") {
-			pending("should start a signal then block on the first value") {
+			it("should start a signal then block on the first value") {
+				var sink: Signal<Int, NoError>.Observer!
+				let producer = SignalProducer<Int, NoError> { observer, _ in
```

Comment:
Can we use `SignalProducer.buffer` instead, to avoid this weird saving of the observer?